wire-ios/Wire Notification Service Extension/NotificationService.swift

```diff
@@ -62,22 +62,66 @@ final class NotificationService: UNNotificationServiceExtension {
     }
 
     private func loadNotificationService() -> NotificationServiceProtocol? {
-        // API version decides which service to use, if we don't have it
-        // yet then we simply supress the notification request.
-        guard let apiVersion = BackendInfo.apiVersion else {
-            WireLogger.notifications.warn("no resolved api version, not loading service")
+        let info = Bundle.appMainBundle.infoDictionary
```

Comment:
This is the top level object for the NSE. Although these environment vars are only needed for the new code, it doesn't hurt to have them here.